uaclient/api/u/pro/security/cves/cve/v1.py

```diff
@@ -1,13 +1,12 @@
 import datetime
```

Comment:
During the rename, this file was left behind. I believe all of `uaclient/api/u/pro/security/cves/cve/*` can be deleted